desktop-widgets/statswidget.cpp

```diff
@@ -93,6 +93,8 @@ StatsWidget::StatsWidget(QWidget *parent) : QWidget(parent)
 	view = qobject_cast<StatsView *>(root);
 	if (!view)
 		qWarning("Oops. The root of the StatsView is not a StatsView.");
+	if (view)
+		view->setVisible(isVisible()); // Start in non-visible state.
```

Comment:
that `setVisible(isVisible())` call confuses me. Why does that make sense?
Especially given the comment? The comment would match a `setVisible(false)` call, right?